src/main/java/io/codelex/arithmetic/practice/CozaLozaWoza.java

```diff
@@ -0,0 +1,30 @@
+package io.codelex.arithmetic.practice;
+
+public class CozaLozaWoza {
+    public static void main(String[] args) {
+
+        int i = 1;
+        while (i < 111) {
+            for (int l = 0; l < 10; l++) {
```

Comment:
šķiet, ka šis for loop līmenis nebija vajadzīgs, jo dēļ `while (i<111)` tāpat tiks izpildīts nākamais for loop `for (int j = 0; j < 11; j++) {...}` arī pēc tam, kad pirmā rindiņa ar 11 vērtībām tiks izdrukāta.

Reply:
Nu tad ņem nost lieko. :) Un šajā uzdevumā būtu jāiztiek ar vienu vienīgu ciklu, bet Tev šobrīd ir trīs - while + divi for loop.

Reply:
Pārtaisīju uzdevumu ar 1 for loop